tools/go-cli/go-whisk-cli/commands/action.go

```diff
@@ -719,7 +719,7 @@ func init() {
     actionCreateCmd.Flags().BoolVar(&flags.action.docker, "docker", false, "treat artifact as docker image path on dockerhub")
     actionCreateCmd.Flags().BoolVar(&flags.action.copy, "copy", false, "treat artifact as the name of an existing action")
     actionCreateCmd.Flags().BoolVar(&flags.action.sequence, "sequence", false, "treat artifact as comma separated sequence of actions to invoke")
-    actionCreateCmd.Flags().StringVar(&flags.action.kind, "kind", "", "the kind of the action runtime (example: swift:3)")
+    actionCreateCmd.Flags().StringVar(&flags.action.kind, "kind", "", "the kind of the action runtime (supported: swift:3, nodejs:6)")
```

Comment:
I was thinking that the CLI help would list the possible supported list of `kind` values.  Is there another way to obtain this list in the format the CLI is expecting?